
# REQUEST
#===============================================================================
Header header

# input point cloud, for which a 2D convex hull should be estimated
sensor_msgs/PointCloud2 pointCloud
---

# RESPONSE
#===============================================================================

geometry_msgs/Polygon convexHull

